### ISSUE TITLE
Build Tests

### ISSUE THREAD

**Issue by yparitcher** [SoloKeysSec/solo#53]
please add stm32l432 builds to the Travis CI as to prevent errors like in (#50) & (#51)

you can add the stm32l432 make file commands to the test make target
and add the venv to the stm32l432 make file (not just the docs) and have it try to mergehex automatically

**Comment by yparitcher** [comment]
also please add a build-debugboot target
```
build-debugboot: clean2 debugboot-boot clean debugboot-app
$(merge_hex) solo.hex bootloader.hex all.hex
```

**Comment by yparitcher** [comment]
fixed in #51
